src/sentry/search/eap/uptime_results/attributes.py

```diff
@@ -7,8 +7,8 @@
     for column in COMMON_COLUMNS
     + [
         ResolvedAttribute(
-            public_alias="trace_id",
-            internal_name="trace_id",
+            public_alias="trace",
+            internal_name="sentry.trace_id",
```

Comment:
Will this break anything else or was this just a fix?

Reply:
Just a fix - we aren't relying on this id anywhere else in the ui, afaik.

Worst case it breaks something only in the sentry org where we have reading from EAP enabled. But I doubt it will even break anything there.